uk.ac.kcl.inf.modelling.xdsml.gemoc_henshin/src/uk/ac/kcl/inf/modelling/xdsml/gemoc_henshin/engine/core/Lang.xtend

```diff
@@ -0,0 +1,12 @@
+package uk.ac.kcl.inf.modelling.xdsml.gemoc_henshin.engine.core;
+/**
+ * code taken from the GEMOC concurrent ccsl engine
+ */
+import org.eclipse.gemoc.execution.concurrent.ccsljavaxdsml.api.extensions.languages.ConcurrentLanguageDefinitionExtensionPoint
+class Lang extends ConcurrentLanguageDefinitionExtensionPoint{
```

Comment:
Do we really need this? Why? If it's needed, can we document why? If it's not needed, can we remove it?

Reply:
It needs to stay. I've updated the comment.